frontend/src/App.tsx

```diff
@@ -479,7 +478,7 @@ function AppHeader() {
             <ApplicationLauncherItem
                 key="ocp_launch"
                 isExternal
-                icon={<OpenshiftIcon style={{ color: '#EE0000' }} />}
```

Comment:
<img width="413" alt="image" src="https://user-images.githubusercontent.com/42188127/172448501-b3bef63e-894f-4c88-a46c-3d85097d24bc.png">
Seems like the colour needs to be specified here.